library/Solarium/QueryType/Select/ResponseParser/Component/Grouping.php

```diff
@@ -67,7 +67,9 @@ public function parse($query, $grouping, $data)
             // parse field groups
             $valueResultClass = $grouping->getOption('resultvaluegroupclass');
             $documentClass = $query->getOption('documentclass');
-            foreach ($grouping->getFields() as $field) {
+
+            // check grouping fields as well as the grouping function (either can be used in the query)
+            foreach (array_merge($grouping->getFields(), [$grouping->getFunction()]) as $field) {
```

Comment:
Currently PHP 5.3 is still supported, so short array syntax must be avoided. The next major Solarium release will update the requirements.